### ISSUE TITLE
password.liquid template

### ISSUE THREAD

**Issue by suture** [Shopify/Timber#298]
I think this is a new one http://docs.shopify.com/themes/theme-development/templates/password-liquid

**Comment by seeReadCode** [comment]
What's the story on this page not being in the Timber repo?

**Comment by ccmeyers** [comment]
+1

**Comment by Cam** [comment]
+1. I was just thinking this yesterday.

**Comment by ccmeyers** [comment]
If you create a `password.liquid` file in your `templates` folder, Shopify will automatically render it. I took the following code from https://docs.shopify.com/themes/development/templates/password-liquid and built on that to customize it.
```
{% unless shop.password_message == blank %}
  {{ shop.password_message }}
{% endunless %}

{% form 'storefront_password' %}

{{ form.errors | default_errors }}

<label for="password">Enter store using password:</label>
<input type="password" name="password">
<input type="submit" value="Enter" class="btn btn-white">

{% endform %}
```

**Comment by seeReadCode** [comment]
I did the same as I imagine a few others have.  That said, it would be nice to have liquid code from their example in the Timber repo.  There are also values that need to be added to the locale file if I remember correctly.

**Comment by ccmeyers** [comment]
I totally agree. And yes, you're right about adding values to the locale file. See pull request #499
